### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by Carreau**
we don't have header cell anymore, 
and don't officially support a specific browser version, support "latest stable"

### CODE REVIEW

File: docs/source/notebook.rst [hunk 1]
```diff
@@ -272,13 +272,12 @@ nbconvert.
...
 
-You can provide a conceptual structure for your computational document as a
-whole using different levels of headings; there are 6 levels available, from
-level 1 (top level) down to level 6 (paragraph). These can be used later for
-constructing tables of contents, etc.  As with Markdown cells, a heading
-cell is replaced by a rich text rendering of the heading when the cell is
-executed.
-
+If you want to provide structure for your document, you can use markdown
+heading.  Markdown headings consists of 1 to 6 hash sign ``#`` followed by a
```
Comment (willingc) [inline docs/source/notebook.rst]: headings. Markdown headings consist of 1 to 6 hash ``#`` signs ...

File: docs/source/notebook.rst [hunk 2]
```diff
@@ -272,13 +272,12 @@ nbconvert.
...
+space and the title of your section. Markdown heading will be converted to
+click able link for each section you can refer to, and will be use as hint for
```
Comment (willingc) [inline docs/source/notebook.rst]: The markdown heading will be converted to a clickable link for a section of the notebook. It is also used as a hint when exporting to other document formats, like PDF. We recommend using only one markdown header in a cell and limit the cell's content to the header text. For flexibility of text format conversion, we suggest placing additional text in the next notebook cell.

**Comment by willingc** [comment]
@Carreau Made a few edits. Use what works for you :smile:

**Comment by Carreau** [comment]
Thanks. 

I added you on my fork, so you can do correction directly if even needed.

**Comment by Carreau** [comment]
(and I incorporated your version)

**Comment by ellisonbg** [comment]
Do we want to unbind the keyboard shortcuts in this release (for heading
cells)?

On Thu, Oct 29, 2015 at 7:44 PM, Matthias Bussonnier <
notifications@github.com> wrote:

> (and I incorporated your version)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/notebook/pull/687#issuecomment-152358369>.
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by Carreau** [comment]
"You shall not break user keyboard shortcut in minor releases"

(-1 in unbinding in general, I, at least, use them extensively)

Envoyé de mon iPhone

**Pull request merged**